src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVCustomReadFilter.java

```diff
@@ -0,0 +1,94 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import org.broadinstitute.hellbender.engine.filters.ReadFilter;
+import org.broadinstitute.hellbender.engine.filters.ReadFilterLibrary;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+/**
+ * Customized serializable reads filter, based on cmd line arguments provided, for structural variation purposes
+ */
+public final class SVCustomReadFilter implements ReadFilter {
```

Comment:
If this class is going to be used in multiple SV tools, then you need to add unit tests for it. Otherwise, make it a private class inside `InsertSizeMetricsCollectorSpark`